src/nvim/globals.h

```diff
@@ -725,7 +725,7 @@ EXTERN bool need_highlight_changed INIT(= true);
 EXTERN FILE *scriptout INIT(= NULL);  ///< Stream to write script to.
 
 // volatile because it is used in a signal handler.
-EXTERN volatile int got_int INIT(= false);  // set to true when interrupt
+EXTERN volatile bool got_int INIT(= false); // set to true when interrupt
                                             // signal occurred
```

Comment:
```suggestion
EXTERN volatile bool got_int INIT(= false); // set to true when interrupt signal occurred
```

Reply:
I guess this `volatile` is also no longer needed as Nvim just terminates on SIGINT.